histomicstk/FeatureExtractionHaralick.py

```diff
@@ -0,0 +1,106 @@
+import skimage.feature
+import numpy as np
+
+
+def FeatureExtractionHaralick(I, Dst=1, Ng=256):
+    """
+    Calculates 26 Haralick features from an image
+    Parameters
+    ----------
+    I : array_like
+        An intensity image
+    Dst : Distance of neighboring pixels in an image. Default value = 1.
+    Ng : Number of gray level co-occurance matrix. Default value = 256.
+    Returns
+    -------
+    H : array_like
+        A matrix containing mean and ranges for 26 features
+    References
+    ----------
+    .. [1] Haralick, et al. "Textural features for image classification,"
+    IEEE Transactions on Systems, Man, and Cybernatics, vol. 6, pp: 610-621,
+    1973.
+    .. [2] Luis Pedro Coelho. "Mahotas: Open source software for scriptable
+    computer vision," Journal of Open Research Software, vol 1, 2013.
+    """
+    # sets the number of angles: 0 45 90 135. this must not be modified.
+    NumberofAngles = 4
```

Comment:
@cdeepakroy Very good point !! I was also thinking about the binary mask to be used as a parameter because this haralick feature computation can be used for cell classification. My first thought was to add this function inside "for region in regionprops(Label)" in FeatureExtraction.py. For example,  
"FeaturesofHaralick[region.label-1, :] = \
            ComputeHaralickFeatures(In[region.bbox[0]:region.bbox[2], region.bbox[1]:region.bbox[3]]) " In this case, the ComputeHaralickFeatures uses the region of a nuclei as a parameter.
But, as you mentioned above, this function could be more independently used for other usages when adding a binary mask as a parameter. I think your approach would be better than my previous thought.

For the feature names, I'll update it soon.

For the angle selection, I have no idea of that yet. How about 1: horizontal and vertical 2: other two angles, 3: all of angles :)